pages/war.php

```diff
@@ -54,6 +54,8 @@ function processWarBattleEnd($battle, $player): string {
     $stat_gain_chance = 100;
 
     $battle_result = "";
+    $patrol_id = $battle->getPatrolId();
```

Comment:
I've been noodling over what we could call these, thinking maybe "Village Patrol" could be the right term, wdyt?
```suggestion
    $patrol_id = $battle->getVillagePatrolId();
```

Reply:
Sounds good to me!